backend/src/main/java/gov/cdc/usds/simplereport/service/sms/TextMessageStatusService.java

```diff
@@ -89,8 +89,8 @@ private List<String> getQueryStringKeys(String queryString) {
   }
 
   private Optional<String> getNumberByMessageId(String messageId, String twilioNumber) {
-    final Integer PREFIX_START = 0;
-    final Integer PREFIX_END = 8;
+    final int PREFIX_START = 0;
```

Comment:
The only reason to use `Integer` over `int` is when you potentially have null values (since primitives can't store null). These ints were never null, so it's cheaper to store `int` instead of `Integer`.

https://github.com/CDCgov/prime-simplereport/security/code-scanning/59